vaas-app/src/vaas/external/tasty_validation.py

```diff
@@ -70,6 +70,12 @@ def form_args(self, bundle):
 
         for field in relation_fields:
             if field in kwargs['data']:
-                kwargs['data'][field] = self.uri_to_pk(kwargs['data'][field])
+                if isinstance(kwargs['data'][field], list):
+                    uris = []
+                    for uri in kwargs['data'][field]:
+                        uris.append(self.uri_to_pk(uri))
+                    kwargs['data'][field] = uris
```

Comment:
it could be simplified by using map function: https://docs.python.org/3/library/functions.html#map

```
kwargs['data'][field] = map(self.uri_to_pk, kwargs['data'][field])
```

or list comprehension

```
kwargs['data'][field] = [self.uri_to_pk(uri) for uri in kwargs['data'][field])
```

Reply:
Nice! I didn't known the map function that I can use like this way :)